src/Microsoft.Health.Fhir.Core/Features/Operations/Export/Models/ExportJobProgress.cs

```diff
@@ -27,12 +29,28 @@ protected ExportJobProgress()
         [JsonProperty(JobRecordProperties.Page)]
         public uint Page { get; private set; }
 
+        [JsonProperty(JobRecordProperties.TriggeringResourceId)]
+        public string TriggeringResourceId { get; private set; }
+
+        [JsonProperty(JobRecordProperties.SubSearch)]
+        public ExportJobProgress SubSearch { get; private set; }
```

Comment:
Would it make sense to call this CompartmentSearch?

Reply:
I went with sub search because this is also needed for Group export when searching for resources in nested Groups. I wanted a name that would be accurate for both scenarios.